src/sagemaker/model.py

```diff
@@ -313,6 +315,12 @@ def __init__(
             MODEL_ENABLE_NETWORK_ISOLATION_PATH,
             sagemaker_session=self.sagemaker_session,
         )
+        self.env = resolve_value_from_config(
```

Comment:
Please evaluate adding unit test coverage for this module?

Reply:
yes, this is covered using a tests:
test_create_model_with_sagemaker_config_injection_with_primary_container
test_create_model_with_sagemaker_config_injection_with_container_defs